.github/workflows/coverage.yml

```
@@ -20,6 +20,7 @@ jobs:
       - name: Prepare
         run: .github/scripts/run-prepare ${{ matrix.buildsystem }} ${{ matrix.os }}
       - name: Test
+        if: github.repository == "jupp0r/prometheus-cpp"
```

There's a parsing error in the green (?!?) coverage check. Maybe the condition has to be surrounded by `${{ }}`
```suggestion
        if: ${{ github.repository == "jupp0r/prometheus-cpp" }}
```